--- before.py	2026-05-13 19:54:39.583450594 +0000
+++ after.py	2026-05-13 19:54:48.846012758 +0000
@@ -182,17 +182,17 @@
            `silent` parameter.
         """
         filename = os.path.join(self.root_path, filename)
-        d = types.ModuleType("config")
-        d.__file__ = filename
+        config_module = types.ModuleType("config")
+        config_module.__file__ = filename
         try:
             with open(filename, mode="rb") as config_file:
-                exec(compile(config_file.read(), filename, "exec"), d.__dict__)
+                exec(compile(config_file.read(), filename, "exec"), config_module.__dict__)
         except OSError as e:
             if silent and e.errno in (errno.ENOENT, errno.EISDIR, errno.ENOTDIR):
                 return False
             e.strerror = f"Unable to load configuration file ({e.strerror})"
             raise
-        self.from_object(d)
+        self.from_object(config_module)
         return True
 
     def from_object(self, obj: object | str) -> None:
